src/launcher/java/org/truffleruby/launcher/options/CommandLineParser.java

```diff
@@ -432,7 +432,7 @@ private void processArgument() throws CommandLineException {
                         config.setOption(OptionsCatalog.VERBOSITY, "true");
                         break FOR;
                     } else if (argument.startsWith("--dump=")) {
-                        LogWithoutTruffle.LOGGER.warning("the --dump= switch is silently ignored as it is an internal development tool");
+                        TruffleLessLog.LOGGER.warning("the --dump= switch is silently ignored as it is an internal development tool");
```

Comment:
yeah I thought about that too, but we would no longer manage all warnings/etc. with logger. So I am in favor of keeping the 2 logger classes.